spec/javascripts/CharacterCountSpec.js

```diff
@@ -0,0 +1,50 @@
+describe("CharacterCount", function() {
+  describe("onReadyCharacterCount", function() {
+    var noCKEditor, changeEditorCount;
+
+    beforeEach(function() {
+      changeEditorCount = spyOn(window, 'changeEditorCount').and.callThrough();
+      noCKEditor = spyOn(window, 'noCKEditor').and.callThrough();
+    });
+
+    describe('noCKEditor', function() {
+      describe('home page', function() {
+        beforeEach(function() {
+          $('body').addClass('pages home')
+          loadFixtures('character_count.html');
+
+          onReadyCharacterCount();
+          $('#moment_why').keyup()
```

Comment:
Add missing semicolon! Though semi-colons are necessary for JS, let's stick with ES6 linting conventions :)
Please update this throughout this document :D